<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>981</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Boton" name="pbUrbanismo" native="true">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>450</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="Boton" name="pbPaisajes" native="true">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>450</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="Boton" name="pbTecnologia" native="true">
   <property name="geometry">
    <rect>
     <x>523</x>
     <y>450</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="Boton" name="pbAnimales" native="true">
   <property name="geometry">
    <rect>
     <x>687</x>
     <y>450</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="Navigator" name="navigatorWidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>1001</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Boton</class>
   <extends>QWidget</extends>
   <header>boton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Navigator</class>
   <extends>QWidget</extends>
   <header>navigator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
